json-rpc/src/methods.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Module contains RPC method handlers for Full Node JSON-RPC interface
+use crate::views::AccountView;
+use anyhow::{ensure, Result};
+use core::future::Future;
+use libra_types::{account_address::AccountAddress, account_config::AccountResource};
+use serde_json::Value;
+use std::{collections::HashMap, convert::TryFrom, pin::Pin, str::FromStr, sync::Arc};
+use storage_client::StorageRead;
+
+type RpcHandler = Box<
+    fn(Arc<dyn StorageRead>, Vec<Value>) -> Pin<Box<dyn Future<Output = Result<Value>> + Send>>,
```

Comment:
What happens when some other handler needs `Arc<dyn SmthElse>` and then `Arc<dyn SmthElse10>`?
Is this fn arg list going to grow to accommodate every use case?

Reply:
eventually we're going have wrapper struct similar to existent AC service that wraps various required fields (e.g. storage client, mempool sender, etc.)